crates/serde/src/json_u256.rs

```diff
@@ -175,6 +179,8 @@ where
 #[cfg(test)]
 mod test {
     use super::JsonU256;
+    #[allow(unused_imports)]
+    use alloc::{vec, vec::Vec};
```

Comment:
https://github.com/alloy-rs/alloy/blob/main/crates/eips/src/eip2930.rs#L7-L8

Reply:
I do this but it do not work.